### PR TITLE
Create ActiveMQ plist, fix #38713

### LINKED ISSUE #38713: missing plist for activemq
A plist file is not generated for activemq. Would be very useful to have one.

### PR DESCRIPTION
**Pull request by southwolf**
Fix #38713. But ActiveMQ use 61613 as broker port, whic might conflict with RabbitMQ and other MQs. There should be some hints or warnings.

### CODE REVIEW

**Comment by xu-cheng** [comment]
```
        Error: 4 problems in 1 formulae
activemq:
 * 14: Trailing whitespace was found
 * 16: Trailing whitespace was found
 * 19: Trailing whitespace was found
 * 40: Trailing whitespace was found
```

**Comment by southwolf** [comment]
@xu-cheng  Sorry, didn't notice that. Fixed now.

File: Library/Formula/activemq.rb [hunk 1]
```diff
@@ -12,6 +12,32 @@ def install
...
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>Program</key>
+        <string>#{opt_bin}/activemq</string>
```
Comment (xu-cheng) [inline Library/Formula/activemq.rb]: I think these two lines are unnecessary. They are duplicated with `ProgramArguments`. Am I right?

**Comment by southwolf** [comment]
Thank you, it is unnecessary to use both `Program` and `ProgramArguments`

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `activemq: add plist.`.

**Pull request closed without merging**